DQMOffline/JetMET/python/jetMETAnalyzer_cfi.py

```diff
@@ -83,87 +83,87 @@
     #
     # For caloMETAnalysis "met"
     #
-     caloMETAnalysis = metDQMParameters.clone(
-         allHist = cms.bool(True)
-     ),
+    # caloMETAnalysis = metDQMParameters.clone(
```

Comment:
Add a comment saying these lines are commented since metDQMParameters does not exist